draft-ietf-quic-transport.md

```diff
@@ -698,18 +702,18 @@ sequence of frames, as described in {{frames}}.
 
```

Comment:
There's confusion here about what "packet" we're talking about. This entire section is about "QUIC packets", but this particular subsection is about UDP datagrams, and not really about QUIC packets. I think we should do away with giving this a name, since we don't otherwise refer to the container UDP datagram as a "simple packet".

I would suggest renaming this section to "Coalescing Packets", and then the rest of the text simply works for the most part (with a few suggestions below).